package.json

```diff
@@ -181,6 +181,7 @@
     "topojson-client": "3.0.0",
     "trunc-html": "1.0.2",
     "trunc-text": "1.0.2",
+    "tslib": "^1.9.3",
```

Comment:
Is this supposed to be a devDependency?

Reply:
Nope, TypeScript will inject require statements for `tslib` when we use things like a `for...of` over an iterable, so `tslib` needs to be installed outside of dev too.

Reply:
Ah, gotcha. Thanks.